app/policies/timelog_policy.rb

```diff
@@ -7,23 +7,23 @@ def initialize(current_user, timelog)
   end
 
   def index?
-    current_user
+    @current_user.admin?
```

Comment:
can you access timelogs from user profile (not admin)? I don't think so. Also, your tests should replicate this logic. If no, you don't need such tests, which are green but don't help you.